pkgs/desktops/gnome-3/core/baobab/default.nix

```diff
@@ -4,13 +4,13 @@
 
 let
   pname = "baobab";
-  version = "3.38.0";
+  version = "40.0";
 in stdenv.mkDerivation rec {
   name = "${pname}-${version}";
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${name}.tar.xz";
-    sha256 = "0ac3fbl15l836yvgw724q4whbkws9v4b6l2xy6bnp0b0g0a6i104";
+    sha256 = "0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
You missed that hash.

Reply:
For some reason the servers are timing out for me.